app/controllers/todo_items_controller.rb

```diff
@@ -1,15 +1,15 @@
 class TodoItemsController < ApplicationController
   def index
     begin
-      @todo_list = TodoList.find_by(id: params[:todo_list_id])
+      @todo_list = TodoList.find(params[:todo_list_id])
     rescue ActiveRecord::RecordNotFound
-      redirect_to new_todo_list_path
+      redirect_to new_todo_list_path and return
```

Comment:
You added this return.. why do you think this might be needed, or in some cases, not needed?